src/mono/wasm/wasm.targets

```diff
@@ -10,7 +10,7 @@
   <ItemGroup>
     <WasmPInvokeModules Include="libSystem.Native"/>
     <WasmPInvokeAssemblies Include="$(MonoArtifactsPath)\System.Private.CoreLib.dll"/>
-    <WasmPInvokeAssemblies Include="$(ArtifactsBinDir)\System.Runtime\$(NetCoreAppCurrent)-Unix-$(Configuration)\System.Runtime.dll"/>
+    <WasmPInvokeAssemblies Include="$(ArtifactsBinDir)\System.Runtime\$(NetCoreAppCurrent)-$(Configuration)\System.Runtime.dll"/>
```

Comment:
This list seems very error-prone. Maybe we can add P2Ps and get the output assemblies? 

cc: @ViktorHofer

Reply:
I can create an issue to track this.